db/migrate/20170707150520_update_container_template_types.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateContainerTemplateTypes < ActiveRecord::Migration[5.0]
+  class ContainerTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Updating type column for ContainerTemplate") do
+      ContainerTemplate.update_all(:type => "ManageIQ::Providers::OpenShift::ContainerManager::ContainerTemplate")
```

Comment:
@lfu Tests are failing because `OpenShift` here does not match the test for `Openshift` (lower case "s").

You want to use "Openshift".  Example: https://github.com/ManageIQ/manageiq-providers-openshift/blob/master/app/models/manageiq/providers/openshift/container_manager.rb#L1